pom.xml

```
@@ -147,7 +147,7 @@
     <guava.version>19.0</guava.version>
     <groovy.version>2.4.11</groovy.version>
     <h2database.version>1.3.166</h2database.version>
-    <hadoop.version>3.1.1-TDP-0.1.0-SNAPSHOT</hadoop.version>
+    <hadoop.version>3.1.1-1.0-SNAPSHOT</hadoop.version>
```

```suggestion
    <hadoop.version>3.1.1-0.0</hadoop.version>
```
Same bellow with Tez and Spark.